testpmd/README.md

```diff
@@ -1,55 +1,130 @@
 
-# Stand-alone-testpmd
+# Testpmd with REST API
 
-Testpmd is commonly used in DPDK performance test.
-It makes sense to have the testpmd seperated from the common tool set container and have its own 
-container image. This allows us to quickly add new capabilities to the testpmd without 
-worrying the update impact to other tools in the common tool set container.
+Intel DPDK sample application testpmd is commonly used in DPDK performance test.
```

Comment:
Add "The" before Intel, "performance test" should probably be "performance testing"

Reply:
ack